### PR TITLE
Some features

### PR DESCRIPTION
**Pull request by MarceliJerzynski**
Linked issues: 

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/app/components/layout/footer/footer.component.html [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <div>
-  <mat-toolbar
-    class="footer"
+  <mat-toolbar class="footer"
     [color]="footerColor">
     {{'FOOTER.PUT' | translate}}
   </mat-toolbar>
+  <mat-toolbar class="footer"
+               [color]="footerColor">
+    <a style="color: gray" href="http://www.put.poznan.pl">www.put.poznan.pl</a>
```
Comment (pietrzakkuba) [inline src/app/components/layout/footer/footer.component.html:8]: `style="color: gray" `
setting style in html? Why not is css?

File: src/app/components/pages/requests/views/rector-panel/rector-panel.component.ts [hunk 2]
```diff
@@ -13,17 +13,17 @@ export class RectorPanelComponent implements OnInit {
...
-  
+
   headers = {
     firstName: {description: 'Imię', type: 'text'},
     surname: {description: 'Nazwisko', type: 'text'},
     instituteName: {description: 'Instytut', type: 'text'},
     country: {description: 'Kraj', type: 'text'},
     city: {description: 'Miasto', type: 'text'},
-    status: {description: 'Status', type: 'text'},
+    statusPl: {description: 'Status', type: 'text'},
```
Comment (pietrzakkuba) [inline src/app/components/pages/requests/views/rector-panel/rector-panel.component.ts:23]: like that change very much

File: src/app/components/shared/security/LocalStorageService.ts [hunk 3]
```diff
@@ -2,27 +2,23 @@ import { Injectable } from '@angular/core';
...
-const request = 'request';
-const status = 'status';
 
 @Injectable({
   providedIn: 'root'
 })
 export class LocalStorageService{
 
-    roleSubject = new BehaviorSubject(this.role);
+    personalDataSubject = new BehaviorSubject(null);
```
Comment (pietrzakkuba) [inline src/app/components/shared/security/LocalStorageService.ts:12]: awesome

File: src/styling/variables.scss [hunk 4]
```diff
@@ -1,10 +1,11 @@
 // layout
 $header-height: 56px;
-$footer-height: 80px;
+$footer-height: 40px;
```
Comment (pietrzakkuba) [inline src/styling/variables.scss:3]: isn't footer to narrow now with just 40 pixels height?

**Comment by pietrzakkuba** [comment]
Nice changes. In future we can consider changing primary, accent and warning colors instead of overriding them in css files. I think it's not too complicated. I will look into that.

File: src/app/components/layout/footer/footer.component.html (hunk 1 above)
Comment (MarceliJerzynski) [reply src/app/components/layout/footer/footer.component.html:8]: Cause Im lazy, hello?

**Review by pietrzakkuba — APPROVED** [review]

**Pull request merged**